frontend/app/components/pageHeader.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { PageHeader } from  'react-bootstrap';
+
+class TitleHeader extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <PageHeader>
+        Ickly Page Header Hot Reloaded!
+      </PageHeader>
```

Comment:
If you change the text in the pageheader and save it in your editor, it should automatically be updated in the browser without a manual reload.